src/race/race_manager.hpp

```diff
@@ -724,6 +724,14 @@ class RaceManager
         return m_minor_mode == MINOR_MODE_TIME_TRIAL;
     }   //  isTimeTrialMode
     // ------------------------------------------------------------------------
+    int currentModeTimePrecision()
```

Comment:
A little comment explaining it's a number of decimals would be welcome, otherwise those 2 - 3 numbers are a bit unclear